cellengine/utils/api_client/APIError.py

```diff
@@ -9,10 +10,10 @@ class APIError(BaseException):
 
     def __str__(self):
         if self.status_code:
-            return "CellEngine API responded with \
+            return re.sub(' +', ' ', "CellEngine API responded with \
```

Comment:
Do you have an example error message where there are extra spaces? Our API error messages sometimes include the user-provided value, e.g.
```
fcsFileId "x    y" is invalid
```
would be unsafe to collapse whitespace in.

Reply:
This is good to know! I do have a situation where there are extra spaces, but I think I can deal with the problem more specifically.